coriolis/tests/test_cache.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2023 Cloudbase Solutions Srl
 # All Rights Reserved.
 
-from unittest import mock
-from coriolis import exception
 from coriolis import cache
+from coriolis import exception
 from coriolis.tests import test_base
+from unittest import mock
```

Comment:
I think the pep8 issue here is actually the fact that there was no free line between Python builtin module imports and third-party ones. Please try the following:

```python
from unittest import mock

from coriolis import cache
from coriolis import exception
```